src/pages/index.js

```diff
@@ -7,96 +7,208 @@ import Card from "../components/Card.js";
-/*                                         Form Validator                                         */
+/*                                         Class Instances                                        */
 /* ---------------------------------------------------------------------------------------------- */
-
 const editProfileFormValidator = new FormValidator(config, profileEditModalSelector);
 const addCardFormValidator = new FormValidator(config, cardModalSelector);
-editProfileFormValidator.enableValidation();
-addCardFormValidator.enableValidation();
+const avatarFromValidator = new FormValidator(config, profileAvatarModalSelector);
+const deleteCardFormValidator = new FormValidator(config, deleteCardModalSelector);
+const userInfo = new UserInfo(profileTitleSelector, profileDescriptionSelector, profileAvatarSelector);
+const editProfilePopup = new PopupWithForm(profileEditModalSelector, handleProfileFormSubmit);
+const previewImagePopup = new PopupWithImage(previewImageModal);
+const addNewCardPopup = new PopupWithForm(cardModalSelector, handleSubmitCard);
+const deleteImagePopup = new PopupWithConfirm(deleteCardModalSelector);
+const editAvatarPopup = new PopupWithForm(profileAvatarModalSelector, handleProfileAvatarSubmit);
+
+const api = new API({
+  baseURL: "https://around.nomoreparties.co/v1/cohort-3-en",
+  headers: {
+    authorization: "3bb6a079-e94c-4226-a104-258379e1896b",
+    "Content-Type": "application/json",
+  },
+});
 
 /* ---------------------------------------------------------------------------------------------- */
-/*                                         Profile Classes                                        */
+/*                                         Form Validators                                        */
 /* ---------------------------------------------------------------------------------------------- */
 
-const userInfo = new UserInfo(profileTitleSelector, profileDescriptionSelector);
+editProfileFormValidator.enableValidation();
+addCardFormValidator.enableValidation();
+avatarFromValidator.enableValidation();
+deleteCardFormValidator.enableValidation();
+/* ---------------------------------------------------------------------------------------------- */
+/*                                           API Promise                                          */
+/* ---------------------------------------------------------------------------------------------- */
 
-const editProfilePopup = new PopupWithForm(profileEditModalSelector, (inputsObject) => {
-  userInfo.setUserInfo(inputsObject.title, inputsObject.description);
-  editProfilePopup.close();
-});
+let userId;
+
+//! Do Not Delete
+Promise.all([api.getInitialCards(), api.getUserInfo()])
+  .then(([initialCards, userData]) => {
+    userId = userData._id;
+    userInfo.setUserInfo(userData.name, userData.about);
+    userInfo.setProfileAvatar(userData.avatar);
+    const cardListSection = new Section(
```

Comment:
Only one instance of class `Section` should be created in the project because there is only **one** section for the cards.

Reply:
Hi, I am very confused. I cannot seem to find a second instance of the class section. When I used cmd + F and type in section, this is the only instance I see. Can you point to other one?

Reply:
> Can you point to other one

Hi)

The other one doesn't exist because in line 154 you added the card not with Section https://github.com/mnunezsa95/se_project_aroundtheus/pull/1#discussion_r1246286757 


So, that's why I put the remark here